docs/source/api.rst

```diff
@@ -62,3 +62,13 @@ Sssp
     :members: sssp
     :undoc-members: sssp
     
+
+Dask
+=========
+
+Pagerank
+-------
+
+.. automodule:: cugraph.dask.pagerank.pagerank
+    :members:
+    :undoc-members:
```

Comment:
This will conflict with the documentation updates (https://github.com/rapidsai/cugraph/pull/412). Had a chance to build a new document? We need to decide how to present SNMG algorithms in the documentation. If you want to merge this faster, we may leave this SNMG routine as undocumented, and resolve this in a separate issue (and discuss what's the proper structure to document SNMG routines there).

And side notes, the number of "==" and "-" under Dask and Pagerank should match the number of characters in Dask and Pagerank (if not, you'll see warnings when you run make html).

Reply:
Yeah, I was trying to follow the syntax on #412, I guess I didn't get all the subtilities yet :). 
I was hoping to merge this today to simplify the development of dask related features to everyone involved. 

Do I understand correctly that you recommend dropping my change in `docs/source/api.rst`, merge both #412 and #411 asap without conflicts, and address the dask update in `docs/source/api.rst` later?

Reply:
Yes, to merge this faster, we may just drop changes in api.rst in this PR. #412 should be merged before we start implementing SNMG documentations.

Reply:
dropped doc in api.rst